### ISSUE TITLE
Plugin Exception: notifications.json not found

### ISSUE THREAD

**Issue by rli** [aws/aws-toolkit-jetbrains#5150]
v3.43

```
com.intellij.diagnostic.PluginException: D:\Users\...\AppData\Local\JetBrains\IntelliJIdea2024.3\aws-static-resources\notifications.json [Plugin: aws.toolkit.core]
	at com.intellij.ide.startup.impl.StartupManagerImplKt$launchActivity$1.invokeSuspend(StartupManagerImpl.kt:502)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:104)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:608)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:873)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:763)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:750)
Caused by: java.nio.file.NoSuchFileException: D:\Users\...\AppData\Local\JetBrains\IntelliJIdea2024.3\aws-static-resources\notifications.json
	at java.base/sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:85)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103)
	at java.base/sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108)
	at java.base/sun.nio.fs.WindowsFileSystemProvider.newByteChannel(WindowsFileSystemProvider.java:234)
	at java.base/java.nio.file.Files.newByteChannel(Files.java:379)
	at java.base/java.nio.file.Files.newByteChannel(Files.java:431)
	at java.base/java.nio.file.spi.FileSystemProvider.newInputStream(FileSystemProvider.java:420)
	at com.intellij.platform.core.nio.fs.DelegatingFileSystemProvider.newInputStream(DelegatingFileSystemProvider.java:72)
	at java.base/java.nio.file.Files.newInputStream(Files.java:159)
	at software.aws.toolkits.core.utils.PathUtilsKt.inputStream(PathUtils.kt:27)
	at software.aws.toolkits.jetbrains.core.notifications.ProcessNotificationsBase.getNotificationsFromFile(ProcessNotificationsBase.kt:42)
	at software.aws.toolkits.jetbrains.core.notifications.ProcessNotificationsBase.retrieveStartupAndEmergencyNotifications(ProcessNotificationsBase.kt:52)
	at software.aws.toolkits.jetbrains.core.notifications.ProcessNotificationsBase._init_$lambda$1(ProcessNotificationsBase.kt:36)
	at software.aws.toolkits.jetbrains.core.notifications.NotificationPollingService.notifyObservers(NotificationPollingService.kt:142)
	at software.aws.toolkits.jetbrains.core.notifications.NotificationPollingService.startPolling(NotificationPollingService.kt:65)
	at software.aws.toolkits.jetbrains.core.notifications.NotificationServiceInitializer.execute(NotificationServiceInitializer.kt:20)
	at com.intellij.ide.startup.impl.StartupManagerImplKt$launchActivity$1.invokeSuspend(StartupManagerImpl.kt:496)
	... 6 more

```

**Comment by rahulsaini** [comment]
Facing same issue with AWS Core 3.43-242 Plugin in the IntelliJ IDEA Community Edition IntelliJ IDEA 2024.2.3 (Community Edition)
Build #IC-242.23339.11, built on September 25, 2024
Runtime version: 21.0.4+13-b509.17 amd64 (JCEF 122.1.9)
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Toolkit: sun.awt.windows.WToolkit
Windows 10.0
GC: G1 Young Generation, G1 Concurrent GC, G1 Old Generation
Memory: 3072M
Cores: 8
Registry:
  ide.experimental.ui=true
  i18n.locale=
Non-Bundled Plugins:
  com.andrey4623.rainbowcsv (2.0.2)
  CMD Support (1.0.5)
  aws.toolkit.core (3.43-242)
  Docker (242.23339.19)
  amazon.q (3.43-242)
  aws.toolkit (3.43-242)
Kotlin: 242.23339.11-IJ

**Comment by rli** [comment]
Fix released in v3.44
